### ISSUE TITLE
[BUG_MP] Wrap bales stops after 2 tries

### ISSUE THREAD

**Issue by MadOtis** [Courseplay/Courseplay_FS22#1908]
**READ FIRST**
https://github.com/Courseplay/Courseplay_FS22/wiki

**Steps to reproduce**
Completed mowing on two fields using CP fieldwork.
Start a new tractor with bale wrapper attached
Select the Bail/wrap CP job with no course specified
Select the "start job" in the menu system.

**Actual behavior**
Tractor drives in a figure-8 loop once or twice, then stops with a message: job complete  But, does not wrap or pick up any bales.

**Expected behavior**
Tractor path-finds bales on field, drives to them individually, then wraps them and moves on to the next one.

**Screenshots**
https://steamcommunity.com/sharedfiles/filedetails/?id=2863519467
https://steamcommunity.com/profiles/76561197971459345/screenshots/

**Courseplay version**
Version used: 7.1.0.3

**Client and Server Logs (GIST)**
https://gist.github.com/MadOtis/12bb3a3ca516bc120968ecdf9cc36c9b

**Player on the server**
2

**Dedicated server or local hosted server**
Dedicated server yes/no ? Yes

**Map**
Calm Lands

**Map fields**
https://gist.github.com/MadOtis/2393d5ac346a7a8edf864013a5308a27

**Comment by schwiti6190** [comment]
@MadOtis Hi there, just asking can these bales be wraped by the bale wraper?
`
2022-09-15 23:41 :30 [dbg4 lp8006] X8.627 VT-Drive/77: SEARCHING_FOR_NEXT_BALE: Found 0 bales.
2022-09-15 23:41 :30 [info lp8006] X8.627 VT-Drive/77: SEARCHING_FOR_NEXT_BALE: No bales found, scan the field once more before leaving for the unload course.
2022-09-15 23:41 :30 [dbg4 lp8006] X8.627 VT-Drive/77: SEARCHING_FOR_NEXT_BALE: Found 0 bales.
`
At least CP thought, that these are not wrapable, maybe these are hay bales ?

**Comment by MadOtis** [comment]
ok, this can be closed.  After much testing and toil, I finally figured out someone else changed the bale size to 1 size larger than the wrapper can handle; this was what was causing the whole issue.  After changing bale size on the baler back to the right size, the next mow/bale/wrap cycle worked as expected.  It just seemed to coincide to us upgrading CP from 7.1.0.2 to 7.1.0.3

TLDR: this can be closed.  Sorry for wasting time.
